types/dockerode/index.d.ts

```diff
@@ -342,7 +342,7 @@ declare namespace Dockerode {
     interface ImageInfo {
         Id: string;
         ParentId: string;
-        RepoTags: string[];
+        RepoTags: string[] | null;
```

Comment:
```suggestion
        RepoTags: string[] | undefined;
```

Can you check this still works, other optional properties use `undefined`

Reply:
Property exists and it's value is null. I think string[] | null  should be used in this case? Undefined is for missing properties. Am I wrong?

Reply:
I've changed type to string[] | null in my node_modules folder and it works fine

Reply:
> Undefined is for missing properties. Am I wrong?

Optional properties are specified using a `?`, e.g. `RepoTags?: string[]` (which should also work).

However, in this case as a precedent has already been set to use `undefined` (see `RepoDigests`), recommend that pattern is followed. Null and undefined are both falsey, so I expect your tests to still work.

Or even:

`RepoTags?: string[] | undefined`

TBF, I don't care that much, it's just good practice to follw existing patterns and not to mix them (I can see from the rest of the file that `undefined` is mainly used, but there are some `null`s and even an `undefined` | `null`)